src/database.ts

```diff
@@ -224,6 +224,14 @@ export interface GetIamPolicyOptions {
   gaxOptions?: CallOptions;
 }
 
+/**
+ * @typedef {object} GetTransactionOptions
+ * * @property {boolean} [optimisticLock] The optimistic lock a
+ *     {@link Transaction} should use while running.
+ */
+interface GetTransactionOptions extends RunTransactionOptions {
+  optimisticLock?: boolean;
```

Comment:
Why do we not want this as a field in `RunTransactionOptions`? Shouldn't it be possible to use this with the normal `runTransaction` method as well?

Reply:
Updated